src/app/views/event/event.component.html

```diff
@@ -2,9 +2,9 @@
   <div class="row">
     <h2 [class.col-9]="user !== null" [class.col-12]="user === null">{{ event.startDate | date : "y" }} {{ event.divisionName ? event.eventName + ' - ' +  event.divisionName + ' Division' : event.eventName }} <span class="badge bg-primary" *ngIf="event.isLive">LIVE</span></h2>
```

Comment:
I know that this wasn't added in this commit, but it should probably have a translation

Reply:
Which words do you mean?

Reply:
"Division" and "LIVE"

Reply:
I don't want to translate the "Division", because all the event name is already in English